src/content/docs/infrastructure/host-integrations/host-integrations-list/redis/sonarqube-integration.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Sonarqube web app monitoring integration
+tags:
+    - New Relic integrations
+    - SonarQube
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your Flutter web application.
+redirects: 
+
+---
+
+import browserFlutterWebDashboard from 'images/browser_screenshot-full_flutter-web-dashboard.png'
+
+Our New Relic Infrastructure agent and New Relic Flex to the instrument and monitor your SonarQube platform. Our SonarQube integration gives you a pre-built dashboard that allows you to evaluate the data from your project, such as line duplications, flaws, and other mistakes in order to diagnose and optimize the code.
+
+<img
+    title="New Relic Flutter web app dashboard"
+    alt="New Relic Flutter web app dashboard"
+    src={browserFlutterWebDashboard}
+/>
+<figcaption>
+After setting up our Flutter integration, we give you a dashboard for your Flutter web app metrics. 
+</figcaption>
+
+## Install [#install]
+
+### Step 1 (optional): Install the infrastructure agent [#infra-install]
+
+Installing the infrastructure agent is optional, but doing this will get you important metrics about the host where your app is running. 
```

Comment:
We tend to want to avoid the word "will". It's a filler word that doesn't add much.

Also, I don't think you need to restate that this is optional. Focus on why someone might want to do this anyway or at least help them make the decision one way or another